tests/mints13/input.dat

```diff
@@ -0,0 +1,30 @@
+molecule mol {
+  N 0.0 0.0 0.0
+  N 0.0 0.0 1.0
+  0 1
+}
+
+basis {
+    assign genbas
+[genbas]
+cartesian
+
+****
+N     0
+S   6   1.00
+      0.4173511460D+04       0.1834772160D-02
+      0.6274579110D+03       0.1399462700D-01
+      0.1429020930D+03       0.6858655181D-01
+      0.4023432930D+02       0.2322408730D+00
+      0.1282021290D+02       0.4690699481D+00
+      0.4390437010D+01       0.3604551991D+00
+SP   3   1.00
+      0.1162636186D+02      -0.1149611817D+00       0g6757974388D-01
```

Comment:
Third column does not look correct

Reply:
Correct, I missed that because mints13 was used as test case in the python subdirectory.